hera_stats/tests/test_stats.py

```diff
@@ -9,25 +9,13 @@
 class Test_Stats():
 
     def setUp(self):
-        self.filepath = os.path.join(DATA_PATH, "onsim.jackknife.spl_ants.Nj20.2018-06-26.17_53_12")
-        self.pc = hp.container.PSpecContainer(self.filepath)
+        filepath = os.path.join(DATA_PATH, "test_pc_jackknife")
+        pc = hp.container.PSpecContainer(filepath)
+        self.jkset = hs.JKSet(pc, "spl_ants")
 
     def test_stats(self):
 
-        hs.stats.anderson(self.pc, summary=True, verbose=True)
-        hs.stats.kstest(self.pc, summary=True, verbose=True)
-
-        p = lambda x: x.imag ** 2
-        hs.stats.anderson(self.pc, proj=p, verbose=True, method="weightedsum")
-        hs.stats.kstest(self.pc, proj=p, bins=10, verbose=True, method="weightedsum")
-
-        hs.stats.avg_spec_with_and_without(self.pc, 1)
-        hs.stats.item_summary(self.pc, 1)
-
-        nt.assert_raises(ValueError, hs.stats.get_pspec_stats, self.pc, sortby=10000)
-        nt.assert_raises(AssertionError, hs.stats.get_pspec_stats, "Thats no moon...")
-
-        nt.assert_raises(AttributeError, hs.stats.standardize, [0,1], [0])
-        nt.assert_raises(AttributeError, hs.stats.standardize, [0], [0])
-        nt.assert_raises(NameError, hs.stats.standardize, [np.arange(10)]*2,
-                         [np.arange(10)]*2, method="It's a space station...")
+        hs.stats.weightedsum(self.jkset[0])
+        hs.stats.zscores(self.jkset[0])
+        hs.stats.anderson(self.jkset[0], summary=True, verbose=True)
```

Comment:
It'd be good to add some tests to make sure these functions give sensible results for simple Gaussian data.